khelo/lib/ui/flow/matches/match_detail/match_detail_tab_view_model.dart

```diff
@@ -81,7 +81,7 @@ class MatchDetailTabViewNotifier extends StateNotifier<MatchDetailTabState> {
             .firstOrNull;
 
         onScorecardExpansionChange(
-          match.matchResult?.teamId ?? match.teams.firstOrNull?.team.id ?? "",
+          match.matchResult?.teamId ?? match.teams.last.team.id ?? "",
```

Comment:
what this change is supposed to do?

Reply:
If there is only one team in the scorecard, then pass the team ID for default expansion.

Reply:
` match.teams.last.team.id ?? ""`
match.teams always have 2 teams, you can use currentplaying teamId for our usecase